src/pyhf/optimize/opt_minuit.py

```diff
@@ -119,20 +117,20 @@ def _minimize(
             if fmin.is_above_max_edm:
                 message += " Estimated distance to minimum too large."
 
-        n = len(x0)
-        hess_inv = default_backend.ones((n, n))
+        hess_inv = None
```

Comment:
Just because I'm not tracking, what's the motivation to have this be `None` by default over the previous `hess_inv = default_backend.ones((n, n))`?

Reply:
Just refactored so that we don't need the `hess_inv` to be a numpy array of ones anymore. We didn't do any calculation using it. And nothing should depend on the hessian always existing.